Update 2024-08-07-community-meeting.mdx (#542)
wasmCloud/wasmCloud.com: community/2024-08-07-community-meeting.mdx

{/* Add summary of topics during the meeting. Each Topic should have an H3 heading. A list is usually sufficient for the recap */}

### DEMO: `wash` and `wadm` developer experience improvements

- As always, notes can only tell so much. Check out the recording for the full demo.
- wasmCloud branch - this is public (prs list).

> - We've changed the was we display tables in `wash` - for longer pieces of information.

Reviewer: ```suggestion
- We've changed the way we display tables in `wash` - for longer pieces of information.
```

- We've also updated the app status command in `wadm` which is a really useful development.

> - If we `run-wash` and `list-apps`, we can see a fresh view. Descriptions are nested in the table underneath. Much easier way to capture this detail.

Reviewer: ```suggestion
- If we run `wash app list`, we can see a fresh view. Descriptions are nested in the table underneath. Much easier way to capture this detail.
```

- If we deploy an app, we did not always get the version back from the app. Now we always get the version back, even if we don't specify it.

> - Our new app status command is well worth a look. In `wash` and `wadm` we're now surfacing the top level status info + all the scaler info for each application.

Reviewer: ```suggestion
- Our new `wash app status` command is well worth a look. In `wash` and `wadm` we're now surfacing the top level status info + all the scaler info for each application.
```
